src/Client/OfficeInterop/OfficeInterop.fs

```diff
@@ -1104,12 +1113,159 @@ let addBuildingBlock (excelTable:Table) (arcTable:ArcTable) (newBB:CompositeColu
 
     loggingList
 
+let prepareTemplateInMemory (tableToAdd:ArcTable) =
+    Excel.run(fun context ->
+        promise {
+
+            let activeSheet = context.workbook.worksheets.getActiveWorksheet()
```

Comment:
Don't we have a "tryGetAnnotationTable" function?

Reply:
fixed